client/src/components/layout/MovieList.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react'
+import MovieTile from "./MovieTile.js"
+
+const MovieList = (props) => {
+    const [movies, setMovies] = useState([])
+
+    const getMovies = async() => {
+        try {
+            const response = await fetch("/api/v1/movies")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            setMovies(body.movies)
+        } catch (error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getMovies()
+    },[])
+
+    const moviesListArray = movies.map((movie)=> {
+        return (
+            <MovieTile movie={movie} key={movie.id}/>
+        )
+    })
+
+    return (
+        <>
+            <h1>All Movies</h1>
+            <ul className="no-bullet">
+                {moviesListArray}
+            </ul>
```

Comment:
nit picking the HTML - since the `moviesListArray` is a collection of `div` elements (as the `div` is the top level element returned for each MovieTile) we should consider replacing the `ul` with a `div`. As an unordered list `ul` will want to have immediate children that are list items `li`